.github/workflows/deploy.yml

```diff
@@ -0,0 +1,38 @@
+name: Deploy
+on:
+  - push
+jobs:
+  deploy:
+    name: Deploy
+    runs-on: ubuntu-latest
+    env:
+      AWS_REGION: eu-west-1
```

Comment:
Don't we need `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY` as environment variables?